core/src/autogluon/core/scheduler/scheduler_factory.py

```diff
@@ -8,11 +8,10 @@
 
 
 _scheduler_presets = {
-    'auto': {'searcher': 'local_sequential_auto'},
-    # 'grid': {'searcher': 'grid'},  # grid commented out as it isn't compatible with most default model search spaces
-    'random': {'searcher': 'random'},
-    'bayesopt': {'searcher': 'bayesopt'},
-    # 'skopt': {'searcher': 'skopt'},  # TODO: Remove skopt? Is it worthwhile to keep as an option?
+    'auto': {'scheduler': 'local', 'searcher': 'bayesopt'},
+    'local': {'scheduler': 'local', 'searcher': 'bayesopt'},
+    'random': {'scheduler': 'fifo', 'searcher': 'random'},
```

Comment:
We may want to consider having all presets be 'local' scheduler. We currently don't have many obvious upsides to using fifo outside of possibly multi-gpu usage for non-advanced users. Additionally, the presets don't allow for more than 1 model being trained in parallel, so that removes any advantage at all fifo would have.

Additionally, this allows us to remove 'local' preset, which seems like a confusing name for it.

Reply:
I also agree 'local' is a confusing name

Reply:
Updated